### PR TITLE
changed log messages to include workflow name as well

### PR DESCRIPTION
**Pull request by junaidHussain-clari**
Pull Request type
----

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes (Please run `./gradlew generateLock saveLock` to refresh dependencies)
- [ ] WHOSUSING.md
- [ ] Other (please describe):

Changes in this PR
----

_Describe the new behavior from this PR, and why it's needed_
Issue #

Alternatives considered
----

_Describe alternative implementation you have considered_

### CODE REVIEW

**Review by hchepey-clari — APPROVED** [review]
LGTM

File: workflow-event-listener/src/main/java/com/netflix/conductor/contribs/listener/archive/ArchivingWithTTLWorkflowStatusListener.java [hunk 1]
```diff
@@ -78,7 +78,7 @@ public void shutdownExecutorService() {
 
     @Override
     public void onWorkflowCompleted(WorkflowModel workflow) {
-        LOGGER.info("Archiving workflow {} on completion ", workflow.getWorkflowId());
```
Comment (ghost) [inline workflow-event-listener/src/main/java/com/netflix/conductor/contribs/listener/archive/ArchivingWithTTLWorkflowStatusListener.java:81]: I would tweak this a little bit.

LOGGER.info("Archiving workflow. Workflow Name : {} Workflow Id : {} Workflow Status : {}", workflow.getWorkflowName(), workflow.getWorkflowId(), <workflow status>);

Comment (junaidHussain-clari) [reply workflow-event-listener/src/main/java/com/netflix/conductor/contribs/listener/archive/ArchivingWithTTLWorkflowStatusListener.java:81]: done

File: workflow-event-listener/src/main/java/com/netflix/conductor/contribs/listener/archive/ArchivingWithTTLWorkflowStatusListener.java [hunk 2]
```diff
@@ -78,7 +78,10 @@ public void shutdownExecutorService() {
...
+        LOGGER.info("Archiving workflow. Workflow Name : {} Workflow Id : {} Workflow Status : {} ",
```
Comment (ghost) [inline workflow-event-listener/src/main/java/com/netflix/conductor/contribs/listener/archive/ArchivingWithTTLWorkflowStatusListener.java]: Should we have each workflow's archival as INFO or DEBUG log?

Comment (junaidHussain-clari) [reply workflow-event-listener/src/main/java/com/netflix/conductor/contribs/listener/archive/ArchivingWithTTLWorkflowStatusListener.java]: We are printing only error msgs in conductor so these logs will not get printed anyway. I have changed the log level to DEBUG in S3 archival class which we will be using. I will change these too.

Comment (junaidHussain-clari) [reply workflow-event-listener/src/main/java/com/netflix/conductor/contribs/listener/archive/ArchivingWithTTLWorkflowStatusListener.java]: Done

**Review by ghost — APPROVED** [review]
LGTM.

**Pull request merged**